lib/gtfsplus/components/GtfsPlusVersionSummary.js

```diff
@@ -43,21 +43,33 @@ export default class GtfsPlusVersionSummary extends Component {
     return (this.props.gtfsplus.timestamp !== this.props.version.fileTimestamp)
   }
 
+  _toggleExpanded = () => {
+    const {gtfsPlusDataRequested, version} = this.props
+    const {expanded} = this.state
+    if (!expanded) gtfsPlusDataRequested(version)
+    this.setState({ expanded: !expanded })
+  }
+
   render () {
     const {
       user,
       version,
-      gtfsPlusDataRequested,
       publishClicked
     } = this.props
     const { feedSource } = version
     const editingIsDisabled = !user.permissions.hasFeedPermission(feedSource.organizationId, feedSource.projectId, feedSource.id, 'edit-gtfs')
     const publishingIsDisabled = !user.permissions.hasFeedPermission(feedSource.organizationId, feedSource.projectId, feedSource.id, 'approve-gtfs')
     const header = (
-      <h3 onClick={() => {
-        if (!this.state.expanded) gtfsPlusDataRequested(version)
-        this.setState({ expanded: !this.state.expanded })
-      }}>
+      <h3
+        onClick={this._toggleExpanded}
+        onKeyDown={(e) => {
```

Comment:
I would say create a ticket and we should consider that in the future but leave it out for now.